src/app/public/modules/vertical-tabset/vertical-tabset-group.component.html

```diff
@@ -11,14 +11,34 @@
     [tabIndex]="-1"
   >
     {{ groupHeading }}
-    <sky-chevron
+    <sky-chevron *ngIf="themeName === 'default'"
       class="sky-vertical-tabset-group-header-chevron"
       [direction]="open ? 'up' : 'down'"
       [disabled]="disabled"
       (directionChange)="toggleMenuOpen()"
       (keyup.enter)="$event.stopPropagation()"
     >
     </sky-chevron>
+    <button *ngIf="themeName === 'modern' && !open"
+      class="sky-btn sky-vertical-tabset-group-header-chevron"
```

Comment:
i think we should be including the sky-btn-icon-borderless class here, may need to adjust the alignment of it.

Reply:
Good call. Done.